### ISSUE TITLE
feat: add swipe actions to item tiles

### ISSUE THREAD

**Issue by gianmarcocalbi** [app-anvil/waste_not#21]
## In scope

This issue is to add the following actions to item tiles:

- swipe right: toggle unwrap item, the item will be flagged as unwrapped (opened) is it was sealed or vice versa.
- swipe left:
	- half way: item has been consumed partially. Show a modal for the user to specify the consumed measure.
	- all the way to the left: the item has been consumed completely, it will be removed from the list.

## Out of scope

Other tile actions like tap or long press.

## Relationships

- Blocked by #17
